allure-framework-modules/mrchecker-webapi-module/src/main/java/com/capgemini/ntc/webapi/pages/httbin/SimpleFormPOSTPage.java

```diff
@@ -0,0 +1,114 @@
+package com.capgemini.ntc.webapi.pages.httbin;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.capgemini.ntc.webapi.core.BasePageWebAPI;
+import com.capgemini.ntc.webapi.core.base.driver.DriverManager;
+import com.capgemini.ntc.webapi.pages.environment.GetEnvironmentParam;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+
+public class SimpleFormPOSTPage extends BasePageWebAPI {
+	
+	private final static String	HOSTNAME	= GetEnvironmentParam.HTTPBIN.getValue();
+	private final static String	PATH		= "/post";
+	private final static String	ENDPOINT	= HOSTNAME + PATH;
+	
+	private static Map<String, Object> requestFields = new HashMap<>();
+	
+	static {
+		requestFields.put("custname", "");
+		requestFields.put("custtel", "");
+		requestFields.put("custemail", "");
+		requestFields.put("comments", "");
+		requestFields.put("delivery", "");
+		requestFields.put("size", "");
+		requestFields.put("topping", "");
+	}
+	
+	public Response submitForm() {
+		return DriverManager.getDriverWebAPI()
+						.contentType(ContentType.URLENC)
+						.formParams(requestFields)
+						.when()
+						.post(ENDPOINT);
+	}
+	
+	@Override
+	public String getEndpoint() {
+		return ENDPOINT;
+	}
+	
+	public void setCustomerName(String customerName) {
+		requestFields.replace("custname", customerName);
```

Comment:
Please validate such scenario: 

1. User in test run new SimpleFormPOSTPage() which  sets    requestFields.put("custname", "").
2. Then in  simpleFormPOSTPage.requestFields.replace("custname", "John")
3. In another step he would like to use one more time simpleFormPOSTPage.requestFields.replace("custname", "Mary").  I thing  this setup will fail, because static{} allows to set this value only once.

Reply:
I have tried to change the name twice with *static{}* and surprisingly it worked fine. However *static*, in this case, is not necessary, so I changed it to a normal one.